plugins/mumble_plugin_win32_ptr_type.h

```diff
@@ -117,16 +132,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
-	pModule=getModuleAddr(modname ? modname : procname);
-	if (!pModule) {
+	hProcess=OpenProcess(PROCESS_QUERY_LIMITED_INFORMATION|PROCESS_VM_READ, false, dwPid);
+	if (!hProcess) {
 		dwPid = 0;
+		pModule = 0;
 		return false;
 	}
 
-	hProcess=OpenProcess(PROCESS_VM_READ, false, dwPid);
-	if (!hProcess) {
+	int result = checkProcessIs64Bit(hProcess);
+	if (result == -1) {
+		return false;
+	}
+
+	// We compare to (not) 0 instead of assigning the value directly to prevent MSVC's C2220 warning:
+	// 'int': forcing value to bool 'true' or 'false' (performance warning)
+	is64Bit = result != 0;
+
+	pModule=getModuleAddr(modname ? modname : procname);
+	if (!pModule) {
 		dwPid = 0;
```

Comment:
Leaking hProcess?

Reply:
Yes, I'm going to move it above `OpenProcess()`.